ZonPHP/pages/top31.php

```diff
@@ -34,7 +34,7 @@
     $paramstr_choose = '?' . $paramstr_choose;
 }
 $footer_display_style = "clear:both; ";
-if (isset($params['hideFooter'])) {
+if ($params['hideFooter'] == true) {
```

Comment:
THANKS... took me 1h to find the GUI bug!

I saw we need to add a closing </div> in all pages, I will add that later